src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -68,7 +66,7 @@ export const GenericTable = ({
   const selectedRowsRef = useRef();
 
   const tableLabels = {
-    label: t(TranslationUtils.getParameterTranslationKey(parameterId), parameterId),
+    label: t(TranslationUtils.getParameterTranslationKey(parameterId), ''),
```

Comment:
I think this change would break the default behavior that displays the parameter id as fallback when no "translated labels" are provided :thinking: